src/Banner.js

```diff
@@ -98,7 +99,8 @@ Banner.propTypes = {
 Banner.defaultProps = {
   bg: 'green',
   textAlign: 'left',
-  showIcon: true
+  showIcon: true,
+  theme: theme
```

Comment:
I don't *think* the composite components need to have the `theme` set as a default prop since the styled components' should have a theme prop already